### PR TITLE
Contributor list when focusing contributor github handle

### PR DESCRIPTION
**Pull request by Bernardstanislas**

### CODE REVIEW

File: frontend/src/pages/ProjectDetails/PaymentActions/PaymentForm/ContributorSelect/View.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ type Props = {
   loading: boolean;
   onChange: ChangeEventHandler<HTMLInputElement>;
   validateContributorLogin: () => boolean | string;
+  contributors: { avatarUrl: string; login: string; id: number }[];
```
Comment (AnthonyBuisset) [inline frontend/src/pages/ProjectDetails/PaymentActions/PaymentForm/ContributorSelect/View.tsx]: Could be nice that this component exposes its own Fragment (to be used in queries and as a type) to specify the data it requires

File: frontend/src/pages/ProjectDetails/PaymentActions/PaymentForm/ContributorSelect/View.tsx [hunk 2]
```diff
@@ -12,27 +14,56 @@ type Props = {
...
+            onChange={onHandleChange}
+            onFocus={() => setOpened(true)}
+            onBlur={() => setOpened(false)}
+            loading={loading}
+          />
+        </Listbox.Button>
+        <div
+          className={classNames({
+            "opacity-0": !opened,
+            "opacity-100": opened,
```
Comment (AnthonyBuisset) [inline frontend/src/pages/ProjectDetails/PaymentActions/PaymentForm/ContributorSelect/View.tsx]: why not completely removing the component from the dom when closed ?

**Review by AnthonyBuisset — APPROVED** [review]

File: frontend/src/pages/ProjectDetails/PaymentActions/PaymentForm/ContributorSelect/View.tsx (hunk 2 above)
Comment (AnthonyBuisset) [reply frontend/src/pages/ProjectDetails/PaymentActions/PaymentForm/ContributorSelect/View.tsx]: nevermind, refactored in a later commit

File: frontend/src/pages/ProjectDetails/PaymentActions/PaymentForm/ContributorSelect/View.tsx (hunk 1 above)
Comment (Bernardstanislas) [reply frontend/src/pages/ProjectDetails/PaymentActions/PaymentForm/ContributorSelect/View.tsx]: For sure!

Comment (Bernardstanislas) [reply frontend/src/pages/ProjectDetails/PaymentActions/PaymentForm/ContributorSelect/View.tsx]: Done

**Pull request merged**